DialServer.js

```diff
@@ -40,6 +40,9 @@ const apps = {
 
       child.on('close', function(code) {
          console.log('closing code: ' + code);
+         child = spawn('npm', ['stop'], {
```

Comment:
unfortunately this will not do anything, as:
- there is no `stop` script 
- you're overwriting the `child` (which was tracking) the spawned process.

A better solution might be just doing `child.kill();` but it'll need to be tested.